### PR TITLE
Hotfix to include more info to genome list

### PR DESCRIPTION
**Pull request by daviortega**
Referent to #19 

2018-10-18-155955_1443x583_scrot

### CODE REVIEW

File: src/app/genomes/genomes-list.pug [hunk 1]
```diff
@@ -23,8 +26,11 @@ mat-table([dataSource]="result" class="genome-table")
         mat-option(value="{{genome.order}}") {{genome.order}}
         mat-option(value="{{genome.family}}") {{genome.family}}
         mat-option(value="{{genome.genus}}") {{genome.genus}}
-  ng-container(matColumnDef="Genbank Version")
-    mat-header-cell(*matHeaderCellDef class="genbank-version-header table-txt-size") Genbank Version
+  ng-container(matColumnDef="Version")
+    mat-header-cell(*matHeaderCellDef class="version-header table-txt-size") Version
+    mat-cell(*matCellDef="let genome" class="version-cell" ) {{genome.version}}
+  ng-container(matColumnDef="GenBank Version")
```
Comment (ToshkaDev) [inline src/app/genomes/genomes-list.pug]: I don't think we need to keep both RefSeq Version and Genebak Version. The later should be removed.

Comment (ToshkaDev) [inline src/app/genomes/genomes-list.pug]: I don't think we need both GenBank Version and RefSeq Version. GenBank Version need to be removed. It's mostly duplication. I'd go with `RefSeq Version` column name, not only `Version`.

Also need to take care that the table looks OK in the smaller screen, i.e. adjust it's minimal width given that the new column (Taxonomy ID) is added.

File: src/app/shop-cart/shop-cart.genomes.list.pug [hunk 2]
```diff
@@ -9,9 +9,15 @@ mat-table([dataSource]="result" class="genome-table")
...
+  ng-container(matColumnDef="Taxonomy ID")
+    mat-header-cell(*matHeaderCellDef class="taxonomy-id-header table-txt-size") Taxonomy ID
+    mat-cell(*matCellDef="let genome" class="taxonomy-id-cell" ) {{ genome.taxonomy_id }}
   ng-container(matColumnDef="Taxonomy")
     mat-header-cell(*matHeaderCellDef class="genome-taxonomy table-txt-size") Taxonomy
     mat-cell(*matCellDef="let genome" class="taxonomy-cell") {{genome.superkingdom}}; {{genome.phylum}}; {{genome.class}}; {{genome.order}}; {{genome.family}}; {{genome.genus}}
+  ng-container(matColumnDef="Version")
+    mat-header-cell(*matHeaderCellDef class="version-header table-txt-size") Version
+    mat-cell(*matCellDef="let genome" class="version-cell" ) {{genome.version}}
   ng-container(matColumnDef="Genbank Version")
```
Comment (ToshkaDev) [inline src/app/shop-cart/shop-cart.genomes.list.pug]: The same here.

**Review by ToshkaDev — CHANGES_REQUESTED** [review]
Left some comments.

File: src/app/genome/genome.view.model.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ export default class GenomeViewModel {
     private taxonomy: any[] = [];
     private mainInfoFields: any[] = [
         {"name": "Organism", "value": "name"},
+        {"name": "NCBI Taxonomy id", "value": "taxonomy_id"},
         {"name": "Assembly level", "value": "assembly_level"}, 
         {"name": "Genbank version", "value": "genbank_version", "ftp_path": "ftp_path"},
```
Comment (ToshkaDev) [inline src/app/genome/genome.view.model.ts]: This line `{"name": "Genbank version", "value": "genbank_version", "ftp_path": "ftp_path"}` also need to be changed to contain RefSeq Version.

**Comment by daviortega** [comment]
Done! @ToshkaDev

**Pull request merged**